### PR TITLE
Documentation of HTTF Lower Plenum Flow Mixing CFD Model

### PR DESCRIPTION
**Pull request by hapfang**
This PR contains the documentation and case files for the Nek5000 CFD case of HTTF lower plenum flow mixing problem. The results CFD investigation is based on the boundary conditions generated by Oregon State University. A continuous investigation is undergoing, which employs another set of boundary conditions proposed by Canadian Nuclear Labs, and an update of the documentation and results will be provided later this year. 
This PR also removes a redundant copy of httf related folder that contains the documentation of HTTF system modeling work.

### CODE REVIEW

File: doc/content/media/httf/lower_plenum_cfd/httf_mesh.png
Comment (GiudGiud) [inline doc/content/media/httf/lower_plenum_cfd/httf_mesh.png:1]: let s reduce this image please, it s bigger than some meshes

File: doc/content/htgr/httf/lower_plenum_cfd.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Nek5000 CFD Modeling of HTTF Lower Plenum Flow Mixing Phenomenon
+
+Contact: Jun Fang, fangj.at.anl.gov 
+
+## Overview
+
+Accurate modeling and simulation capabilities are becoming increasingly important to speed up the development and deployment of advanced nuclear reactor technologies, such as high temperature gas-cooled reactors. Among the identified safety-relevant phenomena for the gas-cooled reactors (GCR), the outlet plenum flow distribution was ranked to be of high importance with a low knowledge level in the phenomenon identification and ranking table (PIRT).
```
Comment (GiudGiud) [inline doc/content/htgr/httf/lower_plenum_cfd.md:7]: do you have a reference for the PIRT?

File: doc/content/htgr/httf/lower_plenum_cfd.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+These phenomena cannot be accurately captured by 1-D system codes; instead, the modeling requires using higher fidelity CFD codes that can predict the temperature fluctuations in the HTTF lower plenum. In this study, a detailed CFD model is established for the lower plenum of scaled, electrically-heated GCR test facility (i.e., High Temperature Test Facility, or HTTF) at Oregon State University. Specifically, the flow mixing phenomenon in HTTF lower plenum is simulated with spectral element CFD software, Nek5000, with the turbulence modeled by the two-equation $k-\tau$ URANS model. The velocity and temperature fields are examined to understand the thermal-fluid physics happening during the lower plenum mixing. As part of the HTTF international benchmark campaign, the simulation results generated from this study will be used for code-to-code and code-to-data comparisons in the near future, which would lay a solid foundation for the use of CFD in GCR research and development.
+
+!alert note
+This documentation assumes that reader has basic knowledge of Nek5000, and is able to run
```
Comment (GiudGiud) [inline doc/content/htgr/httf/lower_plenum_cfd.md:12]:
```suggestion
This documentation assumes that the reader has basic knowledge of Nek5000, and is able to run
```

File: doc/content/htgr/httf/lower_plenum_cfd.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+the example cases provided in [Nek5000 tutorial](https://nek5000.github.io/NekDoc/index.html).
```
Comment (GiudGiud) [inline doc/content/htgr/httf/lower_plenum_cfd.md:13]:
```suggestion
the example cases provided in the [Nek5000 tutorial](https://nek5000.github.io/NekDoc/index.html).
```

File: doc/content/htgr/httf/lower_plenum_cfd.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=subroutine usrdat3() end=subroutine turb_in(wd,tke,tau) include-end=False
+
+The specific inlet temperature and mass flow rates are implemented in +userbc+ with the non-dimensionalized values
+
+!listing htgr/httf/lower_plenum_mixing/httf.usr start=if(id_face.eq.3) end=endif include-end=True
+
+
+## Simulation Results 
+
+The aforementioned CFD cases were carried out on Sawtooth supercomputer located at Idaho National Laboratory. A typical simulation job would use 64 compute nodes with 48 parallel threads per node.
```
Comment (GiudGiud) [inline doc/content/htgr/httf/lower_plenum_cfd.md:96]:
```suggestion
The aforementioned CFD cases were carried out on the Sawtooth supercomputer located at Idaho National Laboratory. A typical simulation job would use 64 compute nodes with 48 parallel threads per node. 
```

Comment (GiudGiud) [reply doc/content/htgr/httf/lower_plenum_cfd.md:96]: is it actually threads or is it processes?
what command are you using to run this? we should include it?

File: doc/content/htgr/httf/lower_plenum_cfd.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+
+## Simulation Results 
+
+The aforementioned CFD cases were carried out on Sawtooth supercomputer located at Idaho National Laboratory. A typical simulation job would use 64 compute nodes with 48 parallel threads per node. 
+A characteristics based time-stepping has been used in the simulations to avoid the limitations imposed by the Courant-Friedrichs-Lewy
+(CFL) number due to the explicit treatment of the non-linear convection
+term. This treatment is necessary due to the existence of small local mesh cells created during the unstructured meshing and tet-to-hex conversion. 
+A maximum CFL number of 1.75 has been reached which corresponds to an average time step size of $2.5\times 10^{-5}$.
+A total simulation time of 5.6 is achieved.
```
Comment (GiudGiud) [inline doc/content/htgr/httf/lower_plenum_cfd.md:101]: 5.6 hours?

File: doc/content/htgr/httf/lower_plenum_cfd.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+## Simulation Results 
+
+The aforementioned CFD cases were carried out on Sawtooth supercomputer located at Idaho National Laboratory. A typical simulation job would use 64 compute nodes with 48 parallel threads per node. 
+A characteristics based time-stepping has been used in the simulations to avoid the limitations imposed by the Courant-Friedrichs-Lewy
+(CFL) number due to the explicit treatment of the non-linear convection
+term. This treatment is necessary due to the existence of small local mesh cells created during the unstructured meshing and tet-to-hex conversion. 
+A maximum CFL number of 1.75 has been reached which corresponds to an average time step size of $2.5\times 10^{-5}$.
+A total simulation time of 5.6 is achieved. 
+Due to the nature of unsteady RANS turbulence model, only a quasi-steady state can be reached. Velocity fluctuations are observed when helium flow enters the lower plenum and passes through the LP posts.
```
Comment (GiudGiud) [inline doc/content/htgr/httf/lower_plenum_cfd.md:102]:
```suggestion
Due to the nature of the unsteady RANS turbulence model, only a quasi-steady state can be reached. Velocity fluctuations are observed when helium flow enters the lower plenum and passes through the LP posts.
```

File: doc/content/htgr/httf/lower_plenum_cfd.md (hunk 5 above)
Comment (GiudGiud) [reply doc/content/htgr/httf/lower_plenum_cfd.md:101]: maybe just say (non dimensional)

but really would make sense to give time in SI

File: doc/content/htgr/httf/lower_plenum_cfd.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+!media httf/lower_plenum_cfd/Uavg.png
+       style=width:60%
+       id=Uavg
+       caption=Time-averaged velocity field (non-dimensional) in the lower plenum.
+
+!media httf/lower_plenum_cfd/Tavg.png
+       style=width:60%
+       id=Tavg
+       caption=Time-averaged temperature field (non-dimensional) in the lower plenum.
```
Comment (GiudGiud) [inline doc/content/htgr/httf/lower_plenum_cfd.md:129]: do you want to add a paragraph to give perspective on this? like where you're going

**Review by GiudGiud — CHANGES_REQUESTED** [review]
looks good

File: doc/content/htgr/httf/lower_plenum_cfd.md (hunk 4 above)
Comment (hapfang) [reply doc/content/htgr/httf/lower_plenum_cfd.md:96]: A batch run script is provided for the related CFD cases (on Sawtooth).

File: doc/content/media/httf/lower_plenum_cfd/httf_mesh.png
Comment (GiudGiud) [inline doc/content/media/httf/lower_plenum_cfd/httf_mesh.png:1]: you ll need to edit the commit 

right now this doesnt remove the 2MB addition, it s still downloaded.

anyhow I ll do it so I can double check testing before merging

**Review by GiudGiud — APPROVED** [review]

File: doc/content/media/httf/lower_plenum_cfd/httf_mesh.png
Comment (hapfang) [reply doc/content/media/httf/lower_plenum_cfd/httf_mesh.png:1]: Thanks Guillaume.

**Comment by GiudGiud** [comment]
Looks great!

Thank you for the contribution @hapfang !

**Pull request merged**